packages/components/src/Link/Link.stories.tsx

```diff
@@ -164,11 +165,18 @@ buttonVariantWithIcon.args = {
   variant: "flat",
   children: (
     <>
-      Link <CheckCircleRoundedIcon purpose="decorative" />
+      Link{" "}
+      <CheckCircleRoundedIcon className={styles.icon} purpose="decorative" />
     </>
   ),
 };
 
+export const buttonVariantWithOnlyIcon = Template.bind(null);
+buttonVariantWithOnlyIcon.args = {
+  children: <CheckCircleRoundedIcon purpose="informative" title="finish" />,
+  variant: "flat",
+};
```

Comment:
Not that I really expect icons to be used in buttons like this.. At the moment, I think most icon-only buttons are using the "text" or "plain" variant.